znvis/cameras/trajectories/zooming_trajectory.py

```
@@ -0,0 +1,114 @@
+    """
+    Class to generate a zooming camera trajectory.
+    The camera zooms from a certain distance to another.
+    """
+
+    def __init__(
+        self,
+        total_frames: int,
+        center: np.ndarray,
+        initial_eye: np.ndarray,
+        zoom_distance: float,
+        up: np.ndarray = np.array([0, 1, 0]),
```

Ruff (0.13.3)

Do not perform function call `np.array` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)